deepmd/tf/utils/network.py

```diff
@@ -289,18 +292,26 @@ def embedding_net(
             stddev=stddev / np.sqrt(outputs_size[ii] + outputs_size[ii - 1]),
             seed=seed if (seed is None or uniform_seed) else seed + ii * 3 + 0,
         )
-        b_initializer = tf.random_normal_initializer(
-            stddev=stddev,
-            mean=bavg,
-            seed=seed if (seed is None or uniform_seed) else seed + 3 * ii + 1,
+        b_initializer = (
+            tf.random_normal_initializer(
+                stddev=stddev,
+                mean=bavg,
+                seed=seed if (seed is None or uniform_seed) else seed + 3 * ii + 1,
+            )
+            if bias
+            else None
         )
         if initial_variables is not None:
             scope = tf.get_variable_scope().name
             w_initializer = tf.constant_initializer(
                 initial_variables[scope + "/matrix_" + str(ii) + name_suffix]
             )
-            b_initializer = tf.constant_initializer(
-                initial_variables[scope + "/bias_" + str(ii) + name_suffix]
+            b_initializer = (
```

Comment:
My idea is this (take the example of TF)

```suggestion
            bias = (scope + "/bias_" + str(ii) + name_suffix) in initial_variables
            b_initializer = (
```